atlasdb-config/src/main/java/com/palantir/atlasdb/factory/DefaultLockAndTimestampServiceFactory.java

```diff
@@ -444,157 +423,6 @@ private static Supplier<InternalMultiClientConjureTimelockService> getMultiClien
                 serviceProvider.getMultiClientConjureTimelockService()));
     }
 
-    private static LockAndTimestampServices createRawLeaderServices(
-            MetricsManager metricsManager,
-            LeaderConfig leaderConfig,
-            Consumer<Object> registrar,
-            Supplier<LockService> lock,
-            Supplier<ManagedTimestampService> time,
-            UserAgent userAgent) {
-        // Create local services, that may or may not end up being registered in an Consumer<Object>.
-        LocalPaxosServices localPaxosServices =
-                Leaders.createAndRegisterLocalServices(metricsManager, registrar, leaderConfig, userAgent);
```

Comment:
There are a bunch of classes here that can now be removed - `Leaders`, `LocalPaxosServices`, `AwaitingLeadershipProxy`, `LocalOrRemoteProxy`, `PingableLeader`, etc.

Reply:
going to delete:
- `Leaders`
- `LocalOrRemoteProxy`

Can't delete because they're used by timelock:
- The entire `com.palantir.leader.proxy` package -> going to remove atlasdb-config's dep on `leader-election-impl` project
- `PingableLeader` -> going to remove atlasdb-config's dep on `leader-election-api` project
- `RemotingClientConfigs` -> going to remove atlasdb-config's dep on `atlasdb-remoting-api`

Reply:
Removing the deps is great, still!

Reply:
I probably should have a look at `atlasdb-remoting-api` (the name of that project does not make sense) when I'm back in